src/main/java/org/gridsuite/modification/server/modifications/VoltageLevelCreation.java

```diff
@@ -0,0 +1,22 @@
+package org.gridsuite.modification.server.modifications;
```

Comment:
copyright header is missing in this new file

Reply:
ok